cadl-tests/src/test/java/com/authentication/union/AzureKeyCredentialPolicy.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.authentication.union;
```

Comment:
Are they same as the other case?

Can we move it to e.g. util package?

Reply:
Yes, they are the same, I will create a util package.